package.json

```diff
@@ -5,7 +5,7 @@
   "main": "app.js",
   "scripts": {
     "start": "env $(sed 's/ # .*//' .env) supervisor app",
-    "test": "npm run lint && jest test",
+    "test": "npm run fix && jest test --runInBand",
```

Comment:
we shouldn't do `fix` here, as it will mean it won't fail on CI when eslint comments are auto-fixable

Reply:
ahhh I didn't realize that the intent was to fail on CI, will update